eng/Versions.props

```diff
@@ -103,10 +103,10 @@
     <MicrosoftDotnetWinFormsProjectTemplates31PackageVersion>4.8.1-servicing.19605.5</MicrosoftDotnetWinFormsProjectTemplates31PackageVersion>
     <MicrosoftDotNetWpfProjectTemplates31PackageVersion>3.1.2-servicing.20066.4</MicrosoftDotNetWpfProjectTemplates31PackageVersion>
     <NUnit3Templates31PackageVersion>1.7.2</NUnit3Templates31PackageVersion>
-    <MicrosoftDotNetCommonItemTemplates31PackageVersion>3.1.10</MicrosoftDotNetCommonItemTemplates31PackageVersion>
+    <MicrosoftDotNetCommonItemTemplates31PackageVersion>3.1.15</MicrosoftDotNetCommonItemTemplates31PackageVersion>
```

Comment:
Oh let me guess: The runtime team didn't need to update their templates between 3.1.10 and 3.1.15❔ If not, this may be wrong.

Reply:
This doesn't come from dotnet/runtime, it comes from dotnet/templating. @vlada-shubina someone from templating should be updating these versions in higher-level branches of dotnet/installer each time one is released

Reply:
We have discussed with @marcpopMSFT and decided not to update the versions as long as there is no change to the package content. And there is no change to the package content (hasn't been for a while).